### DISCUSSION TITLE
"Gestion : directe ou concédée" : pas clair

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1351] [Retours utilisateur·rice]
**Problème :** une utilisatrice (I. L.) m'a fait remarquer que dans le formulaire de création d'une cantine l'alternative "directe ou concédée" pour le mode de gestion n'était pas clair pour elle : en effet, d'après nos critères actuels, son mode de gestion est direct car ses deux cuisines centrales appartiennent à des centres hospitaliers, et non à une société privée (SRC) ; sauf que selon elle, son mode de gestion est concédé car il y a un contrat de concession qui a été signé entre la mairie de la commune (qui gère les cantines des 4 écoles que ses cuisines centrales livrent) et les centres hospitaliers qui détiennent les cuisines centrales.

**Solution proposée :** ajouter une info-bulle à côté de "Mode de gestion" pour préciser notre définition de "concédée".

Votre avis, @valeriemerle et @JenniferStephan?

**Comment by gauthierandre** [comment]
Une cantine qui est satellite, et je crois que c'est le cas dans ton exemple, peut bien être en gestion concédée du fait du contrat de concession qui la lie à une cuisine centrale. Dans le principe, la gestion concédée n'est pas strictement au champs des SRC, le secteur s'étend également aux cuisines centrales. Il faut voir dans le détail, mais dans ce cas, la gestion me semble bien "concédée".
Du coup, il n'y a pas besoin de la petite bulle puisque l'utilisatrice a raison.

**Reply by ghost** [reply]
J'avais tout faux alors. Merci de la rectification, @gauthierandre.
